terraform/modules/data-share-service/alarms.tf

```diff
@@ -123,3 +123,18 @@ resource "aws_cloudwatch_metric_alarm" "queue_process_error_number" {
   }
 }
 
+resource "aws_cloudwatch_metric_alarm" "unconsumed_events" {
+  alarm_name          = "${local.alarm_prefix}-growing-unconsumed-events"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "1"
+  threshold           = "500"
+  alarm_description   = "Number of unconsumed events"
+  treat_missing_data  = "notBreaching"
+  alarm_actions       = [aws_sns_topic.sns_alarm_topic.arn]
+  ok_actions          = [aws_sns_topic.sns_alarm_topic.arn]
+
+  metric_name = "UnconsumedEvents"
+  namespace   = local.metric_namespace
+  period      = "300"
+  statistic   = "Sum"
```

Comment:
If we're reporting hourly (or hourly * number of instances, or once per deployment), does 5 mins make sense?

Reply:
Yeah I was wondering this. I guess if I put it up to an hour the times might not line up too well, but I don't know if that matters? Also making the job hourly in the first place was a bit of a stab in the dark 😅